dkms.conf

```diff
@@ -0,0 +1,7 @@
+PACKAGE_NAME="emlog"
+PACKAGE_VERSION="0.70"
+AUTOINSTALL="yes"
+CLEAN="make clean"
```

Comment:
`make clean` is assumed to be the default, right?
So, can be omitted for brevity (as `make` is), right?
Not a bit deal though.

Reply:
I copied it from nvidia driver, don't know why they wrote this line.

Reply:
> I copied it from nvidia driver, don't know why they wrote this line

Could you, please, try to remove it and see if DMKS works without it?